typings/index.d.ts

```diff
@@ -0,0 +1,53 @@
+declare module 'out-of-band-cache' {
+  type JSONSerializable = Object | string | number | boolean | Date;
```

Comment:
This would be better, since `Date` isn't JSON serializable and `null` is an allowed JSON value:

```typescript
type JSONSerializable = 
  | null 
  | string 
  | number 
  | boolean 
  | { [key: string | number]: JSONSerializable } 
  | Array<JSONSerializable>;
```

Reply:
Great Point, indeed I could have gone little detail with this. 👍

I got the original idea from looking at [lib/index.js:L6](https://github.com/godaddy/out-of-band-cache/blob/master/lib/index.js#L6)

```suggestion
/**
 * @typedef {(Object|String|Number|Boolean|Date)} JSONSerializable
 */
```